extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,129 +18,105 @@
-  times they will cause those contributors to demand the disabling of your
-  [%+ terms.Bugzilla %] account. So, ignore this advice at your peril.
+  It is our intention that [% terms.Bugzilla %] remains a useful tool for reporting 
+  and commenting on [% terms.bugs %], feature-requests, and tasks for the Mozilla community.
 </p>
 
 <p>
-  <a href="https://www.mozilla.org/en-US/about/governance/policies/participation">
-    Please also read the Mozilla Community Participation Guidelines</a>
+  In order to keep [%+ terms.Bugzilla %] a useful, inclusive place for this work, we have 
+  guidelines for using this site which, by using this site, you agree to follow.
 </p>
 
 <p>
-  That said, Mozilla developers are generally a friendly bunch, and will be
-  friendly towards you as long as you follow these guidelines.
+  In addition, your participation on this site is also subject to 
+  <a href="https://www.mozilla.org/about/governance/policies/participation">Mozilla's participation guidelines</a>.
 </p>
 
-<h3>1. Commenting</h3>
-
 <p>
-  This is the most important section.
+  Violations of [% terms.Bugzilla %] etiquette or Mozilla's participation guidelines will be considered 
+  as grounds for suspending your privileges on this site, or suspend your account altogether.
 </p>
 
+<h3>Commenting</h3>
+
 <ol>
   <li>
-  <span class="heading">No pointless comments</span>.
-  Unless you have something constructive and helpful to say, do not add a
-  comment to a [% terms.bug %]. In [% terms.bugs %] where there is a heated debate going on, you
-  should be even more
-  inclined not to add a comment. Unless you have something new to contribute,
-  then the [% terms.bug %] owner is aware of all the issues, and will make a judgement
-  as to what to do. If you agree the [% terms.bug %] should be fixed, vote for it.
-  Additional "I see this too" or "It works for me" comments are unnecessary
-  unless they are on a different platform or a significantly different build.
-  Constructive and helpful thoughts unrelated to the topic of the [% terms.bug %]
-  should go in the appropriate
-  <a href="http://www.mozilla.org/about/forums/">newsgroup</a>.
+    <span class="heading">No abusing people</span>.
+    Constant and intense critique is one of the reasons we build great products. 
+    It's harder to fall into group-think if there is always a healthy amount of
+    dissent. We want to encourage vibrant debate inside of the Mozilla 
+    community, we want you to disagree with us, and we want you to effectively 
+    argue your case. However, we require that in the process, you criticize 
+    <i>things</i>, not <i>people</i>. Examples of things include: interfaces,
+    algorithms, and schedules. Examples of people include: developers,
+    designers and users. <b>Attacking or encouraging attacks on a person 
+    may result in you being banned from [% terms.Bugzilla %]</b>.  
   </li>
```

Comment:
I think this last sentence should be retained, although the appropriate link text is "discussion forum", not "newsgroup".